.circleci/config.yml

```diff
@@ -233,8 +233,6 @@ workflows:
         filters:
           branches:
             ignore: master
-        requires:
-        - test
```

Comment:
Ignore this part, not going to push this.
It's just, for now, I want the integration will run. before implementing all UT